src/plugins/picture/picture-plugin.tsx

```diff
@@ -0,0 +1,83 @@
+import { Button } from "@components/Button";
+import { AnoriPlugin, WidgetConfigurationScreenProps, WidgetRenderProps, WidgetDescriptor } from "@utils/user-data/types";
+import './styles.scss';
+import { translate } from "@translations/index";
+import { useTranslation } from "react-i18next";
+import { Input } from "@components/Input";
+import { useState } from "react";
+import browser from 'webextension-polyfill';
+
+type PicturePluginWidgetConfigType = {
+    url: string
+};
+
+const PictureConfigScreen = ({ saveConfiguration, currentConfig }: WidgetConfigurationScreenProps<PicturePluginWidgetConfigType>) => {
+    const { t } = useTranslation();
+    const [url, setUrl] = useState(currentConfig?.url ?? 'https://source.unsplash.com/random/256x256');
+
+    const onConfirm = () => {
+
+        saveConfiguration({
+            url: url,
+        });
+    };
+
+
+    return (<div className="PictureWidget-config">
+        <div className="field">
+                <label>url:</label>
+                <Input placeholder='https://example.com/image.jpg' value={url} onChange={(e) => setUrl(e.target.value)} />
+            </div>
+
+        <Button className="save-config" onClick={onConfirm}>Save</Button>
+    </div>);
+};
+
+const PicturePlugin = ({ config, instanceId }: WidgetRenderProps<PicturePluginWidgetConfigType>) => {
+    const deerExampleImage = browser.runtime.getURL('/assets/images/picturePreview.jpg')
+    return (<div className="PictureWidget">
+        <img className="Image" src={config.url !== '' ? config.url : deerExampleImage} />
```

Comment:
I don't think we need this deer image at all. I think you can just set src like this `src={config.url}`